User page layout

PilgrimMemoirs opened the pull request:
@mikelikesbikes @ltw

mikelikesbikes commented on app/models/availability.rb:
```diff
@@ -28,11 +28,11 @@ class Availability < ActiveRecord::Base
+      now.utc.to_s,
       utc_eod_for_tz(now,tz).to_s)
   end
 
```
Remove these whitespace/formatting fixes from your PR.

mikelikesbikes commented on app/assets/stylesheets/screen.css.scss:
```diff
@@ -211,10 +219,12 @@ footer {
+
 .right {
-	float: right;
+  float: right;
```
remove this whitespace issue.

mikelikesbikes commented on app/assets/stylesheets/screen.css.scss:
```diff
@@ -152,7 +156,7 @@ footer {
 	text-align: left;
 	margin-bottom: 1em;
-	clear: both;
+  clear: both;
```
remove this whitespace fix.

PilgrimMemoirs commented:
Full screen with 2 columns & user info. Screen width > 720px
![full-screen_2-columns](https://cloud.githubusercontent.com/assets/8316393/5828250/797238c4-a0c7-11e4-8896-fffa2c3dfd7d.png)

Dynamic mobile query to take away floating divs at <720px
![mobile_query_no_float](https://cloud.githubusercontent.com/assets/8316393/5828251/7b119b66-a0c7-11e4-814d-b69846fd6d01.png)

Full screen--dynamically adjusts to absence of user info, appointments
![full_screen_no_info](https://cloud.githubusercontent.com/assets/8316393/5828254/7ce02b2e-a0c7-11e4-9bd1-8ba47b4bf38b.png)

We tried our best, with the help of Matt Dubois, to fix our whitespace errors with git rebase and recommitting hunks we wanted to keep. 

We also added 3 rspec tests to cover the three methods we added to the User model; hopefully test coverage no longer declines.

ltw commented on app/models/user.rb:
```diff
@@ -17,6 +17,18 @@ def self.featured_mentors
     limit(10).order("total_kudos DESC")
   end
 
+  def has_bio
```
I would prefer you to use `bio.present?`. `#present?` returns false for empty arrays, empty strings, and falsy values (nil included).

ltw commented on app/views/users/show.html.erb:
```diff
@@ -1,43 +1,48 @@
+        <a href=<%= href_to_twitters_user(@user) %>><%= "@#{@user.try(:twitter_handle)}" %></a>
+      </p>
+
+      <% if @user.has_bio %>
```
for example
```ruby
if @user.bio.present?
```
